WorldBank-Transport/DRIVER
# Save original record every time a Record is created

## Overview

Extends the model for `RecordAuditLogEntry` and writes the entire record (as a JSON) as well as an MD5 hash signature of the JSON string to it on create. 

### Checklist


## Testing Instructions

 * Create a new record on the UI
 * Verify that for that new record, an audit log entry was created and has a valid `log` and `signature` field
 * On the Django shell, get that `RecordAuditLogEntry` and ensure that `rale.verify_log()` returns `True`
 * Change the `log` field, save the record, and ensure that `rale.verify_log()` returns `False`

Closes [#160206283](https://www.pivotaltracker.com/story/show/160206283)
